tests/src/rules/exports-last.js

```diff
@@ -0,0 +1,117 @@
+import { test } from '../utils'
+
+import { RuleTester } from 'eslint'
+import rule from 'rules/exports-last'
+
+const ruleTester = new RuleTester()
+
+const errors = ['Export statements should appear at the end of the file']
```

Comment:
```js
const error = {
  ruleId: 'exports-last',
  message: 'Export statements should appear at the end of the file',
};
```
You can make this into an array by default but I think you'll need to make an array of multiple errors for some tests.

Would be nice if you could add:
- either the location of the reported node
- the type of the Node : `type: 'ImportDefaultSpecifier`. This might be easier.

At the moment, you are not reporting the export statements, but non-export statements. Any would be fine (though I' prefer reporting the export statements), but the message mentions the export statement, so let's report those. That's why I'd like to add this additional verification.

It could look something like:
```js
const error = type => {
  ruleId: 'exports-last',
  message: 'Export statements should appear at the end of the file',
  type
};

// ------

     test({
        code: `
          export * from './foo'
          const bar = true
        `,
        errors: [error('ExportAllDeclaration')],
      }),
```